src/executor/evaluate/mod.rs

```diff
@@ -161,6 +165,45 @@ async fn evaluate_function<'a, T: 'static + Debug>(
 
             Ok(Evaluated::Value(value))
         }
+        name @ "LEFT" | name @ "RIGHT" => {
+            let convert = |string: String, number: usize| {
+                if name == "LEFT" {
+                    string.get(..number)
+                } else {
+                    let start_pos = if number > string.len() {
+                        0
+                    } else {
+                        string.len() - number
+                    };
+                    string.get(start_pos..)
+                }
+                .or(Some(&string))
+                .unwrap()
+                .to_string()
+            };
+
+            if args.len() != 2 {
+                return Err(EvaluateError::NumberOfFunctionParamsNotMatching {
```

Comment:
Though this is same as `LOWER` & `UPPER`, it has different location so we'd better to have a test case for generating this, too.